hibernate-osgi/src/main/java/org/hibernate/osgi/OsgiPersistenceProvider.java

```diff
@@ -91,8 +91,8 @@ public EntityManagerFactory createContainerEntityManagerFactory(PersistenceUnitI
 		);
 
 		osgiClassLoader.addClassLoader( info.getClassLoader() );
-
-		return Bootstrap.getEntityManagerFactoryBuilder( info, settings, osgiClassLoader ).build();
+		
+		return Bootstrap.getEntityManagerFactoryBuilder( info, settings, osgiClassLoader, new OSGiClassLoaderServiceImpl( osgiClassLoader, osgiServiceUtil ) ).build();
```

Comment:
@kankt, just noticed another piece is missing.  Although you've corrected `createContainerEntityManagerFactory` here, `createEntityManagerFactory` will need a similar fix.  Without it, folks using something like the following will still not be able to use Envers:

```
final ServiceReference serviceReference = bundleContext.getServiceReference( PersistenceProvider.class.getName() );
		final PersistenceProvider persistenceProvider = (PersistenceProvider) bundleContext.getService( serviceReference );
		final EntityManagerFactory emf = persistenceProvider.createEntityManagerFactory( "hibernate-osgi-test", null );
```